app/views/channels.html

```diff
@@ -50,14 +50,35 @@
                     <span ng-if="!channel.priority">-</span>
                   </td>
                   <td data-title="Channel Name">
-                    <a ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.name }}</a>
+                    <a tooltip={{channel.description}} ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.name }}</a>
                     <span ng-if="channel.status=='deleted'">{{ channel.name }}</span>
                   </td>
                   <td data-title="Channel URL Pattern">{{ channel.urlPattern }}</td>
-                  <td data-title="Description">
-                    <a ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.description }}</a>
-                    <span ng-if="channel.status=='deleted'">{{ channel.description }}</span>
+                  
+                  <td data-title="Access">
+                    <div class="channelAccessControlDiv" ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel, 'Access Control')">
+                      
+                      <span ng-if="channel.allow" tooltip="Clients and Roles">
+                        <i class="glyphicon glyphicon-th"></i>
+                        <a ng-repeat="allow in channel.allow"> {{ allow }}</a>
+                      </span>
+                      
+                      <span ng-if="channel.whitelist" tooltip="Whitelisted ip addresses">
+                        <i class="glyphicon glyphicon-ok"></i>
+                        <a ng-repeat="whitelist in channel.whitelist"> {{ whitelist }}</a>
```

Comment:
<?xml version="1.0" encoding="UTF-8"?>
<QBP_Q21
  xmlns="urn:hl7-org:v2xml">
  <MSH>
    <MSH.1>|</MSH.1>
    <MSH.2>^~\&</MSH.2>
    <MSH.3>
      <HD.1>CELLNOSTICS</HD.1>
      <HD.2>www.csir.co.za/msm/cellnostics</HD.2>
      <HD.3>URI</HD.3>
    </MSH.3>
    <MSH.4>
      <HD.1>DOH_STANZA_BOPAPE_II_CLINIC</HD.1>
      <HD.2>www.health.gov.za/stanza-bopape-ii-clinic</HD.2>
      <HD.3>URI</HD.3>
    </MSH.4>
    <MSH.5>
      <HD.1>CSIR_HIE</HD.1>
      <HD.2>hie.meraka.csir.co.za</HD.2>
      <HD.3>URI</HD.3>
    </MSH.5>
    <MSH.6>
      <HD.1>CSIR_MERAKA</HD.1>
      <HD.2>www.csir.co.za/meraka</HD.2>
      <HD.3>URI</HD.3>
    </MSH.6>
    <MSH.7>
      <TS.1>20160426105502+0200</TS.1>
    </MSH.7>
    <MSH.9>
      <MSG.1>QBP</MSG.1>
      <MSG.2>Q22</MSG.2>
      <MSG.3>QBP_Q21</MSG.3>
    </MSH.9>
    <MSH.10>2274308701695773368</MSH.10>
    <MSH.11>
      <PT.1>P</PT.1>
    </MSH.11>
    <MSH.12>
      <VID.1>2.5</VID.1>
    </MSH.12>
    <MSH.21>
      <EI.1>ITI-21</EI.1>
      <EI.2>IHE</EI.2>
    </MSH.21>
  </MSH>
  <QPD>
    <QPD.1>
      <CE.1>Q22</CE.1>
      <CE.2>Get Demographics</CE.2>
      <CE.3>HL7</CE.3>
    </QPD.1>
    <QPD.2>9191919191919091919110909999119</QPD.2>
    <QPD.3>
      <UNKNOWN.1>@PID.3.1</UNKNOWN.1>
      <UNKNOWN.2>901218</UNKNOWN.2>
    </QPD.3>
    <QPD.3>
      <UNKNOWN.1>@PID.3.4.1</UNKNOWN.1>
      <UNKNOWN.2>DOH</UNKNOWN.2>
    </QPD.3>
    <QPD.3>
      <UNKNOWN.1>@PID.3.4.2</UNKNOWN.1>
      <UNKNOWN.2>www.health.gov.za</UNKNOWN.2>
    </QPD.3>
    <QPD.3>
      <UNKNOWN.1>@PID.3.4.3</UNKNOWN.1>
      <UNKNOWN.2>URI</UNKNOWN.2>
    </QPD.3>
    <QPD.3>
      <UNKNOWN.1>@PID.3.5</UNKNOWN.1>
      <UNKNOWN.2>MR</UNKNOWN.2>
    </QPD.3>
    <QPD.3>
      <UNKNOWN.1>@PID.3.6</UNKNOWN.1>
      <UNKNOWN.2>282515</UNKNOWN.2>
    </QPD.3>
  </QPD>
  <RCP>
    <RCP.1>I</RCP.1>
    <RCP.2>
      <CQ.1>10</CQ.1>
      <CQ.2>
        <CE.1>RD</CE.1>
      </CQ.2>
    </RCP.2>
  </RCP>
</QBP_Q21>



>>> Ryan Crichton <notifications@github.com> 26/04/16 11:13 AM >>>
In app/views/channels.html:
 >                      <span ng-if="channel.status=='deleted'">{{
channel.name }}</span>
>                    </td>
>                    <td data-title="Channel URL Pattern">{{
channel.urlPattern }}</td>
> -                  <td data-title="Description">
> -                    <a ng-if="channel.status!='deleted'" href=""
ng-click="editChannel(channel)">{{ channel.description }}</a>
> -                    <span ng-if="channel.status=='deleted'">{{
channel.description }}</span>
> +                  
> +                  <td data-title="Access">
> +                    <div class="channelAccessControlDiv"
ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel,
'Access Control')">
> +                      
> +                      <span ng-if="channel.allow" tooltip="Clients
and Roles">
> +                        <i class="glyphicon glyphicon-th"></i>
> +                        <a ng-repeat="allow in channel.allow"> {{
allow }}</a>
> +                      </span>
> +                      
> +                      <span ng-if="channel.whitelist"
tooltip="Whitelisted ip addresses">
> +                        <i class="glyphicon glyphicon-ok"></i>
> +                        <a ng-repeat="whitelist in
channel.whitelist"> {{ whitelist }}</a>
 Please add commas between each IP
 —
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
     
NaN.     
  
  
--  
This message is subject to the CSIR's copyright terms and conditions,
e-mail legal notice, and implemented Open Document Format (ODF)
standard. 
The full disclaimer details can be found at
http://www.csir.co.za/disclaimer.html.  
This message has been scanned for viruses and dangerous content by
MailScanner,  
and is believed to be clean. 
 
Please consider the environment before printing this email. 
 


-- 
This message is subject to the CSIR's copyright terms and conditions, e-mail legal notice, and implemented Open Document Format (ODF) standard. 
The full disclaimer details can be found at http://www.csir.co.za/disclaimer.html.

This message has been scanned for viruses and dangerous content by MailScanner, 
and is believed to be clean.

Please consider the environment before printing this email.